### PR TITLE
Change legacy backend related + parser docs

### PR DESCRIPTION
**Pull request by TueeNguyen**
## Issue This PR Addresses
Part of #2777 
This PR fixes the docs only

## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [ ] **UI**: Change which improves UI

## Checklist

- [ ] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

**Review by menghif — CHANGES_REQUESTED** [review]
You should remove `legacy monolithic back-end (1.0)` from the "Current Design" section and also change the Telescope version to 3.0:

https://github.com/Seneca-CDOT/telescope/blob/c66bcf485354953f80067350e1a5f1e23c7d20ec/src/web/docusaurus/docs/architecture.md?plain=1#L19

https://github.com/Seneca-CDOT/telescope/blob/c66bcf485354953f80067350e1a5f1e23c7d20ec/src/web/docusaurus/docs/architecture.md?plain=1#L21

File: src/web/docusaurus/docs/api-services/parser.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+### Parser Service
```
Comment (menghif) [inline src/web/docusaurus/docs/api-services/parser.md:1]: Change this `h3` to `h1` to make it the title.

File: src/web/docusaurus/docs/architecture.md [hunk 2]
```diff
@@ -25,22 +25,6 @@ The following gives an overview of the current (i.e. 2.4.0) design of Telescope.
 - autodeployment server
 - test, build, and other automation tooling
 
-### Legacy Monolithic Back-end (1.0)
```
Comment (menghif) [inline src/web/docusaurus/docs/architecture.md:28]: Should we keep some of this to keep some history? Maybe in the "Initial Design" section.

Comment (dbelokon) [reply src/web/docusaurus/docs/architecture.md:28]: I suggest we keep it. When we reach a 4.0 release, we might want to restructure the docs again and decide whether we want to remove or keep some part of it.

File: src/web/docusaurus/docs/architecture.md [hunk 3]
```diff
@@ -52,7 +36,7 @@ The legacy back-end has been split into a series of microservices. Each microser
 - Posts Service (`src/api/posts`) - API for accessing Post and Feed data in Redis (probably not well named at this point)
 - Search Service (`src/api/search`) - API for doing searches against Elasticsearch
 - Status Service (`src/api/status`) - API for accessing Telescope status information, as well as providing the Dashboards
-- Parser Service (`src/api/parser`) - feed and post parsing. Currently disabled, see <https://github.com/Seneca-CDOT/telescope/issues/2111>
+- Parser Service (`src/api/parser`) - feed and post parsing.
```
Comment (dbelokon) [inline src/web/docusaurus/docs/architecture.md:39]: What if we clarify that the Parser service was disabled, and we add a new section for release 3.0 where the parser service was enabled, and the legacy backend was completely removed?

**Comment by TueeNguyen** [comment]
For release 3.0 alpha, we should focus on other feature PRs, this PR on doc changes can go in after. I've moved to 3.0

**Comment by TueeNguyen** [comment]
I've fixed the docs, ready for another round of reviews

File: src/web/docusaurus/docs/api-services/parser.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+---
+sidebar_position: 6
+---
+
+# Parser Service:
+
+The current system uses the parser service in order to run the feed parser and feed queue, see [`src/api/parser/data/feed.js`](https://github.com/Seneca-CDOT/telescope/blob/master/src/api/parser/src/data/feed.js). The blog feeds are stored in Supabase database, they are fetched to be loaded into a [`queue`](https://github.com/Seneca-CDOT/telescope/blob/master/src/api/parser/src/lib/queue.js) as jobs to be processed to create [`Feed`](https://github.com/Seneca-CDOT/telescope/blob/master/src/api/parser/src/data/feed.js) and [`Post`](https://github.com/Seneca-CDOT/telescope/blob/master/src/api/parser/src/data/post.js) objects to be stored in `Redis` (cache) and `Elasticsearch` (indexing) databases, and various microservices use these in order to get their data.
```
Comment (JiaHua-Zou) [inline src/web/docusaurus/docs/api-services/parser.md:7]: Same here

**Review by JiaHua-Zou — APPROVED** [review]

**Review by AmasiaNalbandian — APPROVED** [review]

**Review by sirinoks — APPROVED** [review]

**Review by JiaHua-Zou — APPROVED** [review]

**Review by JerryHue — APPROVED** [review]

**Pull request merged**